### PR TITLE
Feat/entity attributes

### PR DESCRIPTION
**Pull request by amatmv**

### CODE REVIEW

File: pyakeneo/result.py [hunk 1]
```diff
@@ -44,16 +58,16 @@ def get_page_items(self):
...
             r = self._session.get(self._link_next)
-            if r.status_code == 200:
-                (
-                    self._link_first,
-                    self._link_self,
-                    self._link_next,
-                    self._items,
-                    self._count,
-                ) = Result.parse_page(json.loads(r.text))
+            if r.status_code == requests.codes.ok:
```
Comment (EnriqueSoria) [inline pyakeneo/result.py]: `requests` has a function for it

```suggestion
            if is_success(r.status_code):
```

File: pyakeneo/result.py [hunk 2]
```diff
@@ -74,23 +88,46 @@ def get_self_link(self):
     def get_first_link(self):
         return self._link_first
 
-    @staticmethod
-    def parse_page(json_data):
+    @classmethod
```
Comment (EnriqueSoria) [inline pyakeneo/result.py:91]: Why this change if you don't use anything from `cls`?

Comment (amatmv) [reply pyakeneo/result.py:91]: In case some day someone decides to inherit this class and customize that method

File: pyakeneo/result.py (hunk 1 above)
Comment (amatmv) [reply pyakeneo/result.py]: I didn't find the method you suggest, instead I used the ["ok" property](https://requests.readthedocs.io/en/latest/api/#requests.Response.ok)

Comment (EnriqueSoria) [reply pyakeneo/result.py]: oh, sorry, it's from DRF :sweat_smile:

File: pyakeneo/result.py (hunk 2 above)
Comment (EnriqueSoria) [reply pyakeneo/result.py:91]: You can override a staticmethod too, but nevermind

**Review by EnriqueSoria — APPROVED** [review]
